orbmain/src/main/java/com/sun/corba/ee/impl/ior/iiop/IIOPAddressBase.java

```diff
@@ -60,13 +60,15 @@ public boolean equals( Object obj )
             (getPort() == other.getPort()) ;
     }
 
+    @Override
     public int hashCode()
     {
         return getHost().hashCode() ^ getPort() ;
     }
 
+    @Override
     public String toString()
     {
-        return "IIOPAddress[" + getHost() + "," + getPort() + "]" ;
+        return String.format("IIOPAddress[%s, %d]", getHost(), getPort());
```

Comment:
StringBuilder (and now even +) is faster than String.format. Also port formatted with comma-thousand-separator doesn't look nice (depends on system locale).